package Chapter2.`kh-cho`.`khcho-study`.src.main.kotlin.`Exceptions2-8`

fun erroneousCode(){
    val i = "1$".toInt()
}

fun main() {
    erroneousCode()
}